app/src/timebar/components/DatePicker.jsx

```diff
@@ -24,6 +25,7 @@ class DatePicker extends Component {
         </div>
         <ReactDatePicker
           fixedHeight
+          calendarClassName={classnames(['calendar', { endDatePicker: this.props.label === 'end' }])}
```

Comment:
Us-in-6-months will change the label from "end" to "end date" or "finish date" or something else, and we'll have a CSS regression. Best to add a new, more explicit prop, like "calendarClassName"